dallinger/prolific.py

```diff
@@ -281,3 +282,84 @@ def _req(self, method: str, endpoint: str, **kw) -> dict:
             raise ProlificServiceException(json.dumps(error))
 
         return parsed
+
+
+class DevProlificService(ProlificService):
+    """Wrapper that mocks the Prolific REST API and instead of making requests it writes to the log."""
+
+    def _req(self, method: str, endpoint: str, **kw) -> dict:
+        """Does NOT make any requests but instead writes to the log."""
+        self.debug_log(f"method = {method}, endpoint = {endpoint}, kw = {kw}")
```

Comment:
Currently we just log the API call that is made, but I think we should also log the (simulated) response from that API call, if there is any. This can be in a separate log call if you like.

Reply:
Done.

Reply:
Using `debug_log_request` and `debug_log_response` methods now to write to the log.